File: 프로그래머스/0/120902. 문자열 계산하기/문자열 계산하기.java
class Solution {
    public int solution(String my_string) {
        String[] arr = my_string.split(" ");
        int answer = Integer.parseInt(arr[0]);
        int N = arr.length;
        int idx = 1;
        while (idx < N) {
            if (arr[idx].equals("+")) {
                answer += Integer.parseInt(arr[idx+1]);
            } else if (arr[idx].equals("-")) {
                answer -= Integer.parseInt(arr[idx+1]);
            }
            idx++;
        }
        
        return answer;
    }
}

File: 프로그래머스/0/120902. 문자열 계산하기/문자열 계산하기.py
def solution(my_string):
    arr = my_string.split(" ")
    answer = int(arr[0])
    idx = 0
    N = len(arr)
    
    while idx < N:
        if (arr[idx] == "+"):
            answer += int(arr[idx+1])
        elif (arr[idx] == "-"):
            answer -= int(arr[idx+1])
        idx += 1
    return answer